linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -29,8 +33,9 @@ class HttpInitializer extends ProtocolInitializer.Simple {
   }
 
   protected val defaultServer = {
-    val stk = http.ErrorResponder +: Http.server.stack
+    val stk = Http.server.stack
       .replace(HttpTraceInitializer.role, HttpTraceInitializer.server)
+      .prepend(http.ErrorResponder)
```

Comment:
Is this still required on the default server stack, now that it's on the default router stack?  Probably doesn't hurt to include it, but just curious.